.circleci/config.yml

```diff
@@ -83,20 +83,12 @@ commands:
       - run:
           name: Disable Xdebug PHP extension
           command: sudo rm /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
-  install-doctrine-mongodb-odm:
-    steps:
-      - run:
-          name: Install Doctrine MongoDB ODM
-          command: |
-            composer require --prefer-dist --no-progress --no-suggest --no-update --ansi \
-              doctrine/mongodb-odm:^2.0@beta \
-              doctrine/mongodb-odm-bundle:^4.0@beta \
   install-mongodb-php-extension:
     steps:
       - run:
           name: Install mongodb PHP extension
           command: |
-            sudo pecl install mongodb
+            sudo pecl install mongodb-1.5.4
```

Comment:
Why fixing a version?

Reply:
It fails otherwise, as you know pecl is very picky about this. Recently an alpha version has been released, so it stopped working. :laughing:

Reply:
It was only working before this because the last release happened to be the stable version.

Reply:
OK that's too bad :disappointed: